include/GoogleSync/GoogleSync.php

```diff
@@ -740,48 +780,92 @@ public function delEvent($event, $meeting_id)
         $statusCode = $return->getStatusCode();
 
         if ($statusCode >= 200 && $statusCode <= 299) {
-         // 2xx statusCode = success
-            $GLOBALS['log']->debug(__FILE__ . ':' . __LINE__ . ' ' . __METHOD__ . ' - ' . 'Received Success Status Code: ' . $statusCode . ' on delete.');
+            // 2xx statusCode = success
+            $this->logger->debug(__FILE__ . ':' . __LINE__ . ' ' . __METHOD__ . ' - ' . 'Received Success Status Code: ' . $statusCode . ' on delete.');
 
             // This removes the gsync_id reference from the table.
-            $sql = "UPDATE Meetings SET gsync_id = '' WHERE id = '" . $meeting_id . "'";
+            $sql = "UPDATE meetings SET gsync_id = '' WHERE id = {$valMeetingId}";
             $res = $this->db->query($sql);
+            if (!$res) {
+                $this->logger->fatal(__FILE__ . ':' . __LINE__ . ' ' . __METHOD__ . ' - ' . 'Failed to remove gsync_id from record' . $valMeetingId);
+            }
 
             $this->syncedList[] = $meeting_id;
             return true;
         } else {
-            $GLOBALS['log']->fatal(__FILE__ . ':' . __LINE__ . ' ' . __METHOD__ . ' - ' . 'Received Failure Status Code: ' . $statusCode . ' on delete!');
+            $this->logger->fatal(__FILE__ . ':' . __LINE__ . ' ' . __METHOD__ . ' - ' . 'Received Failure Status Code: ' . $statusCode . ' on delete!');
             return false;
         }
     }
 
+    /**
+     * Helper function. Clear all popup reminders from crm meeting
+     *
+     * @param string $event_id The ID of the event in the DB
+     *
+     * @return bool Success/Failure
+     */
+    public function clearPopups($event_id)
+    {
+        if (!isset($event_id) || empty($event_id)) {
+            $this->logger->fatal(__FILE__ . ':' . __LINE__ . ' ' . __METHOD__ . ' - ' . 'Event_id is missing');
+            return false;
+        }
+
+        // Disable all popup reminders for the SuiteCRM meeting
+        $eventIdQuoted = $this->db->quoted($event_id);
+        $sql = sprintf("UPDATE reminders SET popup = '0' WHERE related_event_module_id = %s AND deleted = '0'", $eventIdQuoted);
```

Comment:
Performance: these two sql query could be solved in one step, using only one query and sql request.
see:
https://github.com/salesagility/SuiteCRM/pull/6539/files#diff-034e880ea433e2cc2571088f7cbc6620R817
and:
https://github.com/salesagility/SuiteCRM/pull/6539/files#diff-034e880ea433e2cc2571088f7cbc6620R825

Reply:
Got it.